indico/modules/rb/__init__.py

```diff
@@ -28,6 +29,14 @@
 rb_cache = make_scoped_cache('roombooking')
 
 
+class BookingReasonRequiredOptions(RichIntEnum):
+    __titles__ = [None, _('Always'), _('Never'), _('Not for events')]
+
+    always = 1
+    never = 2
```

Comment:
this is the kind of enum where `never = 0` wouldn't be a bad choice ;) (no strong opinion though, 1-3 is fine too)

Reply:
Fair point. It's changed now.